cesm/flux_atmocn/shr_flux_mod.F90

```diff
@@ -194,6 +195,7 @@ SUBROUTINE flux_atmOcn(logunit, nMax  ,zbot  ,ubot  ,vbot  ,thbot ,   &
     real(R8),intent(out)  ::  qref (nMax) ! diag:  2m ref humidity (kg/kg)
     real(R8),intent(out)  :: duu10n(nMax) ! diag: 10m wind speed squared (m/s)^2
     real(R8),intent(out)  :: ugust_out(nMax) ! diag: gustiness addition to U10 (m/s)
+    real(R8),intent(out)  :: u10res(nMax) ! diag: gustiness addition to U10 (m/s)
```

Comment:
Comment looks wrong:
real(R8),intent(out)  :: u10res(nMax) ! diag:10m resolved wind (no gusts)

Reply:
This is in CESM's flux module.